cmd/erasure-server-pool-decom.go

```diff
@@ -471,18 +474,25 @@ func (z *erasureServerPools) Init(ctx context.Context) error {
 		// time, in future meta.returnResumablePools() might take
 		// '-1' as argument to decommission multiple pools at a time
 		// but this is not a priority at the moment.
-		for _, pool := range meta.returnResumablePools(1) {
-			go func(pool PoolStatus) {
-				switch err := z.Decommission(ctx, pool.ID); err {
-				case errDecommissionAlreadyRunning:
-					fallthrough
-				case nil:
-					z.doDecommissionInRoutine(ctx, pool.ID)
-				default:
-					logger.LogIf(ctx, fmt.Errorf("Unable to resume decommission of pool %v: %w", pool, err))
-				}
-			}(pool)
-		}
+
+		// FIXME: For now there is no auto resume of decom after restart.
```

Comment:
Please rebase don't need to remove this code anymore.

Reply:
done